scripts/check-dialyzer.escript

```diff
@@ -126,7 +126,15 @@ do_warn_path({'beams', Beams}, {N, PLT}) ->
             {N + scan_and_print(PLT, Beams), PLT}
     end;
 do_warn_path({'app', Beams}, {N, PLT}) ->
-    {N + scan_and_print(PLT, Beams), PLT}.
+    try lists:split(5, Beams) of
```

Comment:
This clause should only run for `make dialyze` within an app/core application. I wanted to dialyze the app as one unit so Dialyzer can at least warn when inter-module calls have issues. I expect this to run locally (meaning not in memory-limited CI instances) - did you find this was too much for your dev machine?

Reply:
ci was crashing a lot dialyzing andalso my dev box